Add appveyor.yml for ci-tests on Windows.

larskanis opened the pull request:
The github project needs to be enabled in appveyor, similar to travis-ci. The build and specs runs green, like [here](https://ci.appveyor.com/project/larskanis/msgpack-ruby/build/1.0.3) .

appveyor is slower than travis-ci. That's why I only enabled the most opposed ruby versions, only, for now.

tagomoris commented:
Ruby 1.9.3 is not supported by now.
Please remove it from yml file.

larskanis commented:
With PR https://github.com/msgpack/msgpack-ruby/pull/79 Ruby-1.9.3 could be supported on Windows. It seems to be supported on other platforms as well.

cosmo0920 commented:
Ruby 1.9.3 is already reached EOL. https://www.ruby-lang.org/en/news/2014/01/10/ruby-1-9-3-will-end-on-2015/
It should not be support on Windows.
And Ruby 2.0.0 is reached the state of the security maintenance phase.
We should recommend to use Ruby 2.1 or later, I think.

larskanis commented:
I updated this PR and removed Ruby-1.9.3. Some versions fail, but it can be fixed as in #79.

As a notice: Unfortunately many gems are still not yet prepared for Ruby-2.2 on Windows. This often forces to stay longer on old versions of Ruby on Windows than on other platforms. IMHO gemspec should set a requirement for the ruby version, if the Windows binary gems don't support ruby < 2.0.

tagomoris commented:
Thanks!
